aderm/skywalking
# [test/plugin] immigrate postgresql scenario (#3644)

* Modify plugin test run.sh script parameter error

* shell fun error, delete ()

* delete space line

* [test/plugin] immigrate postgresql scenario

* add licene

* format expectedData.yaml & update support-version.list

* change workload from 3 to 1

* add startup.sh

* update startup.sh

* Add verify health-check segment

* remove health-check segment

* update code. 1. context is not effective; 2.format code. 3.change log to slf4j;

* update expectedData.yaml & change log to log4j2

* Add stage case count

* add immigrate postgresql-9.4-scenario

* repair module name & package name

* repair all postgresql-9.4 to postgresql-above9.4.1207

* change module name to postgresql-above9.4.1207

* change stage name

Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] New feature provided

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
